### DISCUSSION TITLE
retrieve data encrypted with MINIO_SSE_MASTER_KEY

### DISCUSSION THREAD

**Discussion by cmacheret** [minio/minio#14074] [Q&A] [answered]
Hi,

We have an old Minio Version (2019-08-21T19:40:07Z) running doing server side encryption with the environment variable MINIO_SSE_MASTER_KEY. This is the baremetal verison running on some Ubuntu 18.04.

Unfortunately the daemon is not starting anymore, I'm getting this error : 
_ERROR Unable to initialize config system: json: cannot unmarshal object into Go struct field serverConfigV33.region of type string
      > Please ensure a valid value in the configuration file
      HINT:_

I've also tried to run a newer verison of Minio but I'm unable to decrypt the data... Is this variable still supported on newer version of Minio?

Any help would be much appreciated... Any tips and tricks to retrieve the encrypted data would be great!

Thanks a lot in advance& BR
Carole

**Comment by harshavardhana** [comment, marked as answer]
> We have an old Minio Version (2019-08-21T19:40:07Z) running doing server side encryption with the environment variable MINIO_SSE_MASTER_KEY. This is the baremetal verison running on some Ubuntu 18.04.

That is too old @cmacheret you will have to purge the `.minio.sys/config` path and restart all the services.

**Reply by cmacheret** [reply]
Thanks a lot @harshavardhana it works like a charm! (deleted the mentioned files and I was able to restart the (very) old version of minio!
